Thu, 09 Jan 2020 12:06:01 - INFO - Connect to DB
Thu, 09 Jan 2020 12:06:01 - INFO - TRIZMA DB import
Thu, 09 Jan 2020 12:06:01 - INFO - 123TV DB import
Thu, 09 Jan 2020 12:07:18 - INFO - Connect to DB
Thu, 09 Jan 2020 12:07:18 - INFO - TRIZMA DB import
Thu, 09 Jan 2020 12:07:18 - INFO - 123TV DB import
Thu, 09 Jan 2020 12:09:58 - INFO - Connect to DB
Thu, 09 Jan 2020 12:09:58 - INFO - TRIZMA DB import
Thu, 09 Jan 2020 12:09:58 - INFO - 123TV DB import
Thu, 09 Jan 2020 12:16:35 - INFO - Connect to DB
Thu, 09 Jan 2020 12:16:35 - INFO - TRIZMA DB import
Thu, 09 Jan 2020 12:16:36 - INFO - Failed on TRIZMA DB import at: 2020-01-09_11-59-44, for following calling party: 497121580124, for agent id: trz_bmaksimovic, with error:(pyodbc.DataError) ('22001', '[22001] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]String or binary data would be truncated. (8152) (SQLExecDirectW)')
[SQL: insert into records values('CC2Servicenummer','Belgrad','Nicht übermittelt','trz_bmaksimovic',                                  '497121580124','','Nicht übermittelt','497121580124_Saturday, December 28, 2019_5:30:56 PM')]
(Background on this error at: http://sqlalche.me/e/9h9h)
Thu, 09 Jan 2020 12:16:36 - INFO - 123TV DB import
Thu, 09 Jan 2020 12:21:53 - INFO - Connect to DB
Thu, 09 Jan 2020 12:21:53 - INFO - TRIZMA DB import
Thu, 09 Jan 2020 12:21:54 - INFO - 123TV DB import
Thu, 09 Jan 2020 12:22:23 - INFO - Connect to DB
Thu, 09 Jan 2020 12:22:23 - INFO - TRIZMA DB import
Thu, 09 Jan 2020 12:22:23 - INFO - 123TV DB import
